src/public/services/AnalyticsService.ts

```diff
@@ -1,11 +1,11 @@
 import axios from 'axios'
 
-import { AnalyticStats } from 'src/types/analytics'
+import { AnalyticStatsDto } from 'src/types/analytics'
```

Comment:
Our typescript config is wonky,  we still can't import using baseUrl (maybe it's worth a fix soon LOL) as JS compilation will fail. No errors here since it is a type import (and thus won't get compiled to js), but should still take care to use relative imports for now?